code/_compile_options.dm

```diff
@@ -106,10 +106,10 @@
 #endif
 
 #if defined(OPENDREAM)
-#error Compiling BeeStation in OpenDream is unsupported due to BeeStation's dependence on the auxtools DLL to function.
+#warn Compiling BeeStation in OpenDream is unsupported due to BeeStation's dependence on the auxtools DLL to function.
```

Comment:
```suggestion
#error Compiling BeeStation in OpenDream is unsupported due to BeeStation's dependence on the auxtools DLL to function.
```
OD will always fail at runtime so we might as well keep its message as an error.